OctaneSdk/Connector/Credentials/APIKeyConnectionInfo.cs

```diff
@@ -25,18 +25,14 @@ namespace MicroFocus.Adm.Octane.Api.Core.Connector
 	/// </summary>
 	public class APIKeyConnectionInfo : ConnectionInfo
     {
+        public string access_token { get; set; }
```

Comment:
Please explain why you need to enrich this class with additional properties.
Please consider creation new class with additional properties.